### ISSUE TITLE
Trading Fee

### ISSUE THREAD

**Issue by CryptNZ** [peatio/peatio#420]
Hi again,

Is a market fee actually taken (from markets.yml) I cannot see it logged anywhere or is this to be implemented still?

I understand it is taken in a percentage, but I cannot see in the logs or database where it takes the fee?

**Comment by lgn21st** [comment]
Thank you for ask, we'll fix it in next major release, it would be not to let you wait too lang.

**Comment by CryptNZ** [comment]
Thanks for your reply, my next question is;

Are the fees actually taken at the moment? I know they aren't shown but are they still taken if I set it in markets.yml?

**Comment by valuead** [comment]
Read documentation and post 

All this questions were asked million times. 

Getting ridicules.

**Comment by CryptNZ** [comment]
I have read the documentation, but it doesn't specify if the fee is actually taken. I know it is not shown anywhere but nowhere does it say if the fee is actually taken from the trade.

**Comment by valuead** [comment]
No fees taken. System works fees free 

You need  write your own module

**Comment by arruah** [comment]
CryptNZ did you solve this issue?

**Comment by CryptNZ** [comment]
It requires some customization, if you know how to code its not too hard otherwise you will have to wait for an update from the dev's to incorporate it.

**Comment by MrDunne** [comment]
@CryptNZ its considered polite to share changes to open source code with others, even if it was on your own fork.

**Comment by CryptNZ** [comment]
@MrDunne Yes but even we haven't fully implemented it yet, we don't see the point sharing half finished/working work.
